packages/bigquery/test/index.js

```diff
@@ -143,14 +143,38 @@ describe('BigQuery', function() {
       assert(bq instanceof Service);
 
       var calledWith = bq.calledWith_[0];
-
       var baseUrl = 'https://www.googleapis.com/bigquery/v2';
-      assert.strictEqual(calledWith.baseUrl, baseUrl);
+      assert.strictEqual(bq.baseUrl, baseUrl);
       assert.deepEqual(calledWith.scopes, [
         'https://www.googleapis.com/auth/bigquery'
       ]);
       assert.deepEqual(calledWith.packageJson, require('../package.json'));
     });
+
+    it('should not be considered custom endpoint if default', function() {
+      assert.strictEqual(bq.customEndpoint, false);
+    });
+
+    it('should use options.apiEndpoint if defined', function() {
```

Comment:
We don't need this test here, because it's not functionality performed by the code in `bigquery/src/index.js`.